SparkPost/2web2ui
# EO-801 Switch over to Alerts v1 API for Alerts List page

### What Changed
 - Created action/reducer/selector for alerts v1.
 - Added a formatted date and sort key in the selector to modify the list of alerts. 
 - Changed response params from labs/alerts -> v1.  EX:  'alert_metric'->'metric', 'enabled'->'muted'
 - Sorted by last trigger by using a sort key. Also added logic to only show last 4 triggered alerts in the cards panel.

### How To Test
 - Enable user account option `alerts`. This is already enabled in appteam staging.
 - See [here](https://github.com/SparkPost/2web2ui/blob/master/docs/feature-flags.md) to enable user options. Alternatively, [point your local upstreams to staging](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams) and login to our appteam, CID 107 account
 - Edit your openresty configs to contain 
```
    location /api/v1/alerts {
      proxy_pass http://sp-alerts-api;
      include cors;
    }
```
under `server:80`,   `location /api/v1`
 - Navigate to `/alerts-new`.
 - Make sure everything works and that alerts can be toggled and deleted. 

### To Do
- [ ] Possible fix the toggle button bug here.
